Exiled.CustomItems/API/Features/CustomItem.cs

```diff
@@ -999,7 +999,11 @@ protected virtual void OnUpgrading(API.EventArgs.UpgradingItemEventArgs ev)
         /// Called anytime the item enters a player's inventory by any means.
         /// </summary>
         /// <param name="player">The <see cref="Player"/> acquiring the item.</param>
-        protected virtual void OnAcquired(Player player) => ShowPickedUpMessage(player);
+        protected virtual void OnAcquired(Player player) // TODO: Male field 'displayMessage' parameter
```

Comment:
What does this TODO means?

Reply:
If add a parameter to method all overridden method will stop working

Reply:
Yeah I see, just confused about you used the word Male

Reply:
It means to make the displayMessage field into a parameter of the method

Reply:
> Yeah I see, just confused about you used the word Male
> 
> 

Sry for my English ;)